doc/user_guide/howto_create_new_dialect.md

```diff
@@ -0,0 +1,75 @@
+# How to Create a New Dialect of the Document Files Virtual Schema
+
+This guide helps you to create your custom dialect.
+
+## Check if this is the correct guide
+
+First of all, check if you really need a dialect.
+A dialect implements support for an external file provider like BucketFs or S3.
+A dialect does not implement support for different file types see the [user_guide](user_guide.md).
+
+Also, check that you really want a new dialect of the files virtual schema. 
+If your data source is not a file storage, but for example a database, 
+you probably do not want to create a dialect of this Virtual Schema,
+but of the [virtual-schema-common-document](https://github.com/exasol/virtual-schema-common-document).
+
+ ## Getting started
+ 
+ To get started with a new dialect create a new repository and copy the contents of the [bucketfs dialect](https://github.com/exasol/bucketfs-document-files-virtual-schema).
```

Comment:
It's a bad idea to ask somebody copy something. Please, change  this part somehow